developers/blobstream-rollups.md

```diff
@@ -0,0 +1,123 @@
+---
+description: Learn how to build rollups that use Blobstream
+---
+
+## Optimistic rollups
+
+One type of rollups that can be built with Blobstream is optimistic rollups.
+An optimistic rollup is a rollup that commits optimistically to a set of blocks,
+and allows the other parties to verify that the blocks are valid,
+and if they're not, they can create fraud proofs to signal that.
+
+Through the use of Blobstream, Celestia allows optimistic rollups to use Celestia as a DA layer,
+i.e., posting all the rollup data to Celestia, and only send commitments to the settlement contract.
+Then,
+users can use Blobstream as a source of truth to verify the rollup data
+and create fraud proofs in case of a misbehavior.
+
+To build an optimistic rollup that uses Celestia as a DA layer, the following constructions can be inspired by.
+
+### Optimistic rollups that uses a sequence of spans
+
+One way to construct an optimistic rollup that uses Celestia as a DA layer is to post the rollup data in Celestia.
+Then in the rollup header, use the following information to reference a sequence of spans, aka a data pointer:
```

Comment:
in the intro, I would first go into the details of a span if its used for both. Below we reference this span section, but its non-intuitive, since zk rollup devs aren't going to read the optimistic rollup section